### PR TITLE
mail-confirm

### PR DESCRIPTION
**Pull request by nguyenhathanh2212**
mail-confirm

### CODE REVIEW

**Comment by nguyenhathanh2212** [comment]
ready

File: app/Http/Controllers/Ecommerce/CheckoutController.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    use Cart;
+
+    protected $productRepository;
+    protected $orderRepository;
+
+    public function __construct(
+        ProductInterface $productRepository,
+        OrderInterface $orderRepository
+    )
+    {
```
Comment (trangkarol) [inline app/Http/Controllers/Ecommerce/CheckoutController.php]: convention

File: app/Http/Controllers/Ecommerce/CheckoutController.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        $this->productRepository = $productRepository;
+        $this->orderRepository = $orderRepository;
+    }
+    public function index()
+    {
+        try {
+            if (Auth::check()) {
```
Comment (trangkarol) [inline app/Http/Controllers/Ecommerce/CheckoutController.php]: check login hay chua, sao e khong dung middlawre

**Comment by trangkarol** [comment]
commented

**Comment by nguyenhathanh2212** [comment]
ready

**Pull request merged**